src/plugins/discover/public/application/components/discover_grid/discover_grid.tsx

```diff
@@ -133,8 +134,12 @@ export interface DiscoverGridProps {
   useNewFieldsApi: boolean;
 }
 
+const styles = css`
+  background: coral;
+`;
+
```

Comment:
looking forward for `emotion` (😆 , 😢 , :anger)

Reply:
Just a test. I have a to-do item to remove this:

> Remove experimental css and Emotion styles from UI components (Chrome, Canvas)